h2o-logger/build.gradle

```diff
@@ -0,0 +1,17 @@
+plugins {
+    id 'java'
+}
+
+group 'ai.h2o'
```

Comment:
remove these two lines, this is done automagically

Reply:
fixed.